### ISSUE TITLE
Fix GitHub action latest tag

### ISSUE THREAD

**Issue by Alkarex** [FreshRSS/FreshRSS#3664]
https://github.com/FreshRSS/FreshRSS/pull/3524 does not seem to work https://github.com/FreshRSS/FreshRSS/actions/runs/912631064

**Comment by math-GH** [comment]
@Alkarex Is it still an issue?

**Comment by Alkarex** [comment]
Yes. I am tagging `latest` on git manually after a release at the moment
